AdtSampleApp/SampleFunctionsApp/ProcessDTRoutedData.cs

```diff
@@ -38,8 +36,8 @@ public static async Task Run([EventGridTrigger] EventGridEvent eventGridEvent, I
             // Authenticate on ADT APIs
             try
             {
-                ManagedIdentityCredential cred = new ManagedIdentityCredential(adtAppId);
```

Comment:
What is the advantage to use the DefaultAzureCredential in this case, given that we know that in a Function we will want to use managed? For consistency?

Reply:
Both consistency as well as debuggability on a dev box or local execution scenario for testability. I would assume that these samples are going to be used as building blocks and not ready to ship components, so optimizing for good devx I think is key. Aside from that, the Identity team seems to encourage the use of DefaultAzureCredential as well for the generic use case.